tests/test_storage.py

```diff
@@ -210,6 +210,23 @@ def test_storage(storage_connector, table_type, clear_dynamically_created_models
         conn.size() == 2
     ), f"Expected 1 record, got {conn.size()}: {conn.get_all()}"  # expect 2, since storage connector filters for agent1
 
+    # test: update
+    # NOTE: only testing with messages
+    if table_type == TableType.RECALL_MEMORY:
```

Comment:
@sarahwooders decided to only run these tests on Messages but it should still work fine on archival if the string asserts are updated